lib/engine/g_1822/company.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../company'
+
+module Engine
+  module G1822
+    class Company < Company
+      attr_reader :type, :removed, :header_name, :header_color, :header_text_color
+
+      def initialize(game, **opts)
+        @type = opts[:type]&.to_sym || :private
+        @header_name = opts[:header_name] || 'PRIVATE COMPANY'
+        @header_color = game.registered_color(opts[:header_color]) || :yellow
+        @header_text_color = opts[:header_text_color] || :black
+        @removed = false
+
+        super(**opts)
```

Comment:
Recommend calling super first, so that your instance vars don't accidently get overridden.

Reply:
I know that's not the case now, but if someone adds one of those types to the base class, it will be an issue.

Reply:
Sure i can do that. But i have looked and super is really all over the place in the code.